lib/init/grass.py

```diff
@@ -168,14 +168,14 @@ def try_remove(path):
         pass
 
 
-def clean_env(gisrc):
```

Comment:
The idea for the parameter here is that this is one of the functions which should eventually go to grass.script.setup to be (re)used when a session is started using the grass package from Python. (Maybe less important now with grass-session package for external uses of G, but still very useful for subsession dealing with different projections or mapsets.)